content/introducing-user-roles-authorization-with-supertokens/index.md

```diff
@@ -0,0 +1,181 @@
+---
+title: "Introducing User Roles Authorization with SuperTokens"
+description: "With user roles, you can now use SuperTokens to easily attach a different set of permissions to each user."
+date: "2022-10-27"
+cover: "user_roles_cover.png"
```

Comment:
the size of this image in card_covers folder is too large compared to the size in which it's viewed. The card cover folder is only used for the blog listing as opposed to the static/covers folder which is used for when the blog is clicked on.

So make the size of the card_covers image smaller